pkg/agent/openflow/packetin.go

```diff
@@ -63,10 +63,10 @@ func (c *client) RegisterPacketInHandler(packetHandlerReason uint8, packetHandle
 		klog.Errorf("Invalid controller to handle packetin.")
 		return
 	}
-	if c.packetInHandlers[packetHandlerReason] == nil {
-		c.packetInHandlers[packetHandlerReason] = map[string]PacketInHandler{}
+	if c.featureNetworkPolicy.packetInHandlers[packetHandlerReason] == nil {
```

Comment:
Why place packetInHandlers under featureNetworkPolicy?

Reply:
Moved it back.